src/app/ace-utils.ts

```diff
@@ -1,4 +1,5 @@
 declare let ace: any;
+declare let $: any;
```

Comment:
What does this have to do with Angular CLI tools?

Reply:
We have to do this here to stop the tests from failing.

The tests were failing because this value appeared to be undefined. This is because of change of the karma config file.